ppa/archive/templates/archive/snippets/thumbnail_snippet.html

```diff
@@ -0,0 +1,16 @@
+<div style="float:left; width: 50%">
```

Comment:
Please add a comment block at the top here explaining briefly what this is for and what variables it expects.